tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -516,6 +502,10 @@ private void getToolTestResults(Tool tool) {
                                         .asList(date, toolversion.getId(), tag, runner, pipelineNode.getDisplayName(), result, duration,
                                                 logURL);
                                 report.printAndWriteLine(record);
+                                if (result.equals("SUCCESS")) {
+                                    S3Client s3Client = new S3Client();
+                                    s3Client.createObject(toolId, tag, pipelineNode.getDisplayName(), runner, logContent, epochStartTime);
+                                }
                             } catch (NullPointerException e) {
                                 LOG.warn(e.getMessage());
```

Comment:
I would log the exception itself as well. And this should probably be error, e.g., `LOG.error(e.getMessage(), e)`;